### PR TITLE
746 - Bug fix: Adding child node makes it appear like a grandchild node

### LINKED ISSUE #746: Adding child node makes it appear like a grandchild node

### PR DESCRIPTION
**Pull request by narendrasss**
Closes #746 

Changes the default depth to 3 rather than `maxDepth + 1` so that nodes made off an empty Tapestry wouldn't appear as grandchild nodes. I believe this also better reflects the old functionality in `tapestry.js`.

### CODE REVIEW

**Comment by seanRong** [comment]
Looks good to me, definitely prefer this implementation

**Comment by yungalyx** [comment]
nice fix!

File: templates/vue/src/components/TapestryDepthSlider.vue [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default {
   methods: {
     ...mapMutations(["updateNode"]),
     setDefaultDepth() {
-      this.currentDepth = this.settings.defaultDepth || this.maxDepth + 1
+      this.currentDepth = this.settings.defaultDepth || 3
```
Comment (wynnset) [inline templates/vue/src/components/TapestryDepthSlider.vue:100]: `3` should be stored as a constant in the app or at minimum as a data variable. It's not a good idea to hard-code values in the code.

**Review by wynnset — CHANGES_REQUESTED** [review]
Looks good just a small nit

**Comment by narendrasss** [comment]
@wynnset I updated the implementation to fill the default depth with the constant if it's undefined in settings. This should also fix the other issue I encountered in #624.

**Review by wynnset — APPROVED** [review]

**Pull request merged**